opencti-platform/opencti-front/src/private/components/settings/decay/DecayRuleValidator.ts

```diff
@@ -0,0 +1,14 @@
+import * as Yup from 'yup';
+
+const decayRuleValidator = (t: (value: string) => string) => Yup.object().shape({
+  name: Yup.string().min(2).required(t('This field is required')),
+  description: Yup.string().nullable(),
+  active: Yup.boolean(),
+  order: Yup.number().min(1).max(Number.MAX_SAFE_INTEGER),
+  decay_lifetime: Yup.number().min(1).max(Number.MAX_SAFE_INTEGER),
+  decay_pound: Yup.number().min(0).max(Number.MAX_SAFE_INTEGER),
```

Comment:
I'm unsure we should check such case actually (that's a path I'm not sure we want to follow... there is so much place to do so). IMO it is a platform for "educated" people, we can't check for all impossible values.

Reply:
yes I agree with you @labo-flg, I'm going to push a fix with a more reasonable check.

Reply:
... and (max for business) is different from (allowed max), which can also be different depending on where we llok at the stack (like @JeremyCloarec found with elastic).

If we have real business rules (for instance a confidence is always between 0 and 100) we might think about adding form validation (+ backend check, let's not forget API users).

Reply:
I've changed the validation to have max values that make sense, but there aren't any specific check in the backend for min or max values.

Reply:
Your changes look good enough to me 👍
For backend, it might also be excessive checking... 
Actually it's a broader issue. If there are business rules to enforce, we might as well enforce them through the schema - for instance a `min` and `max` options for numeric attributes. Something we might think of later, definitely not here :)